octopusdeploy_framework/resource_project_group.go

```diff
@@ -107,7 +118,9 @@ func (r *projectGroupTypeResource) Update(ctx context.Context, req resource.Upda
 	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
 }
 
-func (r *projectGroupTypeResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+func (r *projectGroupTypeResource) Delete(
```

Comment:
:shakes-fist-at-go-fmt: 
This was a linter auto-commit from GoLand. Let me know if you'd like me to partially revert so there's not a bunch of formatting changes in this PR :)